database/factories/ModelFactory.php

```diff
@@ -76,6 +78,13 @@
     ];
 });
 
+$factory->define(Setting::class, function ($faker) {
```

Comment:
> Seems a bit random to add this in an unrelated pull request...

If I don't do this for the functional tests, they won't run on TravisCI. See [StatusPageControllerTest.php](https://github.com/yoyosan/Cachet/blob/d08bea95e0ed93bab2ff1f4d1b29f25c65d00d8c/tests/Http/Controllers/StatusPageControllerTest.php) or [DashboardControllerTest.php](https://github.com/yoyosan/Cachet/blob/d08bea95e0ed93bab2ff1f4d1b29f25c65d00d8c/tests/Http/Controllers/Dashboard/DashboardControllerTest.php).

It's also used here: ![image](https://cloud.githubusercontent.com/assets/32405/18409881/b8acf440-775b-11e6-9135-4ebbd7483c3c.png)

> Doesn't the cachet:seed command do this anyway?

Yes, it does it, but not the test env.